package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "azure-sdk-for-net",
+  "version": "1.0.0",
+  "dependencies": {
+    "autorest": "^3.7.1"
```

Comment:
can we add a follow up task to bump this version during autorest publish ci https://dev.azure.com/azure-sdk/internal/_build?definitionId=1375 and link it here for tracking

Reply:
Added https://github.com/Azure/azure-sdk-for-net/issues/42614